src/react-components/preferences-screen.js

```diff
@@ -484,6 +490,8 @@ const preferenceLabels = defineMessages({
   }
 });
 
+preferenceLabels.automaticOutputGainControl.helpLink = "https://github.com/mozilla/hubs/issues/3214";
```

Comment:
@brianpeiris As I explained in Discord, I couldn't come up with other ways to extend `preferenceLabels`. Do you know more appropriate way?

Reply:
Ah, I guess `defineMessages` removes extraneous properties. We could consider generalizing it a bit more if we ended up using this pattern more often, but this works for now.
One thing we should consider though is how this would work on Hubs Cloud. If we want Hubs Cloud instances to be completely white-label, it might not make sense for a help link to point to our github repo.
I'd also suggest, to make it more user-friendly, that we create and point to a page under https://hubs.mozilla.com/docs/, but maybe we can do that once we consider this a permanent feature and not an experiment.

Reply:
For Hubs Cloud, one option might be to use the existing "show_docs_link" feature flag, so if this preference does land on Hubs Cloud, at least users will have some ability to disable it. We could also add a new "auto_gain_control_help_link" config so that Hubs Cloud users can decide if they want to change this URL.

I also wonder what benefit you want users to get from linking to that particular issue. Maybe it should link to a new issue or a new discussion with less technical detail about the feature?

Reply:
> One thing we should consider though is how this would work on Hubs Cloud. If we want Hubs Cloud instances to be completely white-label, it might not make sense for a help link to point to our github repo.

Good point. I overlooked it.

> For Hubs Cloud, one option might be to use the existing "show_docs_link" feature flag, so if this preference does land on Hubs Cloud, at least users will have some ability to disable it. We could also add a new "auto_gain_control_help_link" config so that Hubs Cloud users can decide if they want to change this URL.

Maybe it would be a good option. But I don't want to make the configuration more complex for this new feature. So I'm thinking of removing the linking.

> I also wonder what benefit you want users to get from linking to that particular issue. Maybe it should link to a new issue or a new discussion with less technical detail about the feature?

I wanted users to know what Dynamic Audio Normalization is from the linking. But it has been renamed to Automatic incoming voice volume control so perhaps they can imagine what it is from the name now. And I want them to send the feedback for this new feature's quality but I'm thinking of an alternate way of the linking for example writing a blog post instead.